<?xml version="1.0" ?>
<sdf version="1.7">
  <model name="pharmacist_dock">
    <static>true</static>
    
    <link name="base_link">
      <pose>0 0 0.0845 0 0 0</pose>
      
      <inertial>
        <mass>50.0</mass>
        <inertia>
          <ixx>1.0</ixx>
          <ixy>0.0</ixy>
          <ixz>0.0</ixz>
          <iyy>1.0</iyy>
          <iyz>0.0</iyz>
          <izz>1.0</izz>
        </inertia>
      </inertial>
      
      <collision name="collision">
        <geometry>
         <mesh>
              <uri>file:///home/meditab/car_ws_new/src/hey_agv_new/urdf/pharmacist_dock_v2.gltf</uri>
            </mesh>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      
      <visual name="visual">
        <geometry>
          <mesh>
              <uri>file:///home/meditab/car_ws_new/src/hey_agv_new/urdf/pharmacist_dock_v2.gltf</uri>
            </mesh>
        </geometry>
        <material>
          <ambient>0.8 0.8 0.8 1.0</ambient>
          <diffuse>0.8 0.8 0.8 1.0</diffuse>
          <specular>0.2 0.2 0.2 1.0</specular>
        </material>
      </visual>
    </link>

    <link name='apriltag0_link'>
        <visual name='apriltag_visual'>
          <geometry>
            <mesh>
              <uri>file:///home/meditab/car_ws_new/src/hey_agv_new/urdf/tag0.glb</uri>
            </mesh>
          </geometry>
        </visual>
        <collision name='apriltag_collision'>
          <geometry>
            <mesh>
              <uri>file:///home/meditab/car_ws_new/src/hey_agv_new/urdf/tag0.glb</uri>
            </mesh>
          </geometry>
        </collision>
          <pose>-0.434789924621582 -0.09 0.05 3.14 0 0</pose>
      </link>
  </model>
</sdf>